templates/base.html

```diff
@@ -3,7 +3,7 @@
   <head>
     {% load static %}
     <link rel="stylesheet" href="{% static 'styles.css' %}" />
-    <title>socialDISTRIBUTION</title>
+    <h1>Social Distribution</h1>
```

Comment:
```suggestion
    <title>Social Distribution</title>
```
I don't think H1's are supposed to be in the HTML header 😅

Reply:
Fixed